tests/test-cases/Group3-Docker-Compose/3-04-Docker-Compose-Fixtures.robot

```diff
@@ -0,0 +1,419 @@
+# Copyright 2016-2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  Test 3-03 - Docker Compose Basic
+Resource  ../../resources/Util.robot
+Suite Setup  Install VIC Appliance To Test Server  certs=${True}
+Suite Teardown  Cleanup VIC Appliance On Test Server
+
+*** Test Cases ***
+Compose basic
+    ${yml} =  Set Variable  version: "2"\nservices:\n${SPACE}web:\n${SPACE}${SPACE}image: python:2.7\n${SPACE}${SPACE}ports:\n${SPACE}${SPACE}- "5000:5000"\n${SPACE}${SPACE}depends_on:\n${SPACE}${SPACE}- redis\n${SPACE}redis:\n${SPACE}${SPACE}image: redis\n${SPACE}${SPACE}ports:\n${SPACE}${SPACE}- "5001:5001"
+
+    Set Environment Variable  COMPOSE_HTTP_TIMEOUT  300
```

Comment:
I think `Set Environment Variable` persists for the entire suite, so we only need to do this once in this file. See `3-03-Docker-Compose-Basic.robot` for example.